### ISSUE TITLE
bond-cni pod reboot

### ISSUE THREAD

**Issue by ognjen011** [k8snetworkplumbingwg/bond-cni#128]
Could anyone help me understand if this is a normal behaviour. I setup  deployment two pods as a test. I have two network interfaces ens8f1np1 and ens8f0np0 with VF's enabled. I setup a bond-cni and i see two interfaces are created in the pods including bond0 interface. Now when i shut down one physical link everything is working but after about 45s and than  pods redeploy/restart. I dont want   the pods  to redeploy since the other VF should be enabled in via the bond interface if i understand this. 

the other option i would like to try is active-actve mode 

  {
      "cniVersion": "0.3.1",
      "name": "sriov-dpdk-bond",
      "type": "bond",
      "name": "bond1",
      "mode": "802.3ad",
      "failOverMac": 0,
      "linksInContainer": true,
      "miimon": "100",
      "mtu": 1500,
      "links": [
         {"name": "net1"},
         {"name": "net2"}
      ],

I also see pods redploy  when i use 802.3ad

**Comment by SchSeba** [comment]
The bond-cni for sure is not the one that reset the pod.
you need to check if the nic you move to down is not the k8s API communication nic and your node goes down.

is this the only pod that gets redeploy? or others in the same node?

**Comment by ognjen011** [comment]
Yes i did bring it down to test a failover I thought when it fails over to second nic it can be done without redeployment of a pod. Thanks for the help. I decided a better option is to use ovs and achieve VF failover that way without redeployment.

**Comment by SchSeba** [comment]
ok great!

@ognjen011 can we close this issue?

**Comment by ognjen011** [comment]
Yes we can close it. Thanks again.

**Comment by youhangwang** [comment]
@ognjen011 Hi, have you successfully used the bond-cni plugin to set up bond4 in a container? I create two pods, each one have two vfs passthrough from host, and create bond4 in each pod. However, these two pods can't connect to each other.

Here is our net

```
apiVersion: "k8s.cni.cncf.io/v1"
kind: NetworkAttachmentDefinition
metadata:
  name: bond-net1
spec:
  config: '{
  "type": "bond",
  "cniVersion": "0.3.1",
  "name": "bond-net1",
  "mode": "802.3ad",
  "xmitHashPolicy": "layer3+4",
  "lacpRate": "fast",
  "faileOverMac": 1,
  "linksInContainer": true,
  "miimon": "100",
  "mtu": 1500,
  "links": [
     {"name": "net1"},
     {"name": "net2"}
  ],
  "ipam": {
    "type": "host-local",
    "subnet": "10.56.217.0/24",
    "routes": [{
      "dst": "0.0.0.0/0"
    }],
    "gateway": "10.56.217.1"
  }
}'
```

**Comment by ognjen011** [comment]
Hi i think i had a similar issue but my main use case was to shut down one nic without rebooting the pod  in the  end i decided not to use and use  so i didn't spend the time to troubleshot further.
